test/data/css/tests.js

```diff
@@ -230,6 +230,19 @@ exports.test_data = {
             { input: '#bla, #foo{\n\tcolor:black;{{separator_input}}\tfont-size: 12px;\n}', output: '#bla,\n#foo {\n\tcolor: black;{{separator_output}}\tfont-size: 12px;\n}' }
```

Comment:
It seems the original code didn't preserve new lines there anyway. It only preserves new lines between, and within rules, not in the selector list.